src/Stryker.Core/Stryker.Core.UnitTest/Reporters/ReporterFactoryTests.cs

```diff
@@ -12,7 +12,7 @@ public class ReporterFactoryTests
     {
         public static IEnumerable<object[]> getParameters()
         {
-            yield return new object[] {new StrykerOptions("", "Console", "", 1000, null, "debug", false, 1, 80, 60, 0), typeof(BroadcastReporter)};
+            yield return new object[] {new StrykerOptions(), typeof(BroadcastReporter)};
```

Comment:
Different loglevel due to default values.

Reply:
Yes. But does it meter here?